### DISCUSSION TITLE
Normal spy for local chat

### DISCUSSION THREAD

**Discussion by anon228-1337** [Zrips/CMI#3591] [Feature Requests]
https://github.com/Zrips/CMI/issues/3510
https://github.com/Zrips/CMI/issues/3260
https://github.com/Zrips/CMI/issues/3444

when it will be added?

**Comment by CloudeLecaw** [comment]
Duplicated other requests - close

It will be added when its time for this.

**Comment by anon228-1337** [comment]
up
